src/main/resources/db/migration/V037__product_categories_change.sql

```diff
@@ -0,0 +1,32 @@
+DROP MATERIALIZED VIEW IF EXISTS offline_merchant;
+DROP MATERIALIZED VIEW IF EXISTS online_merchant;
+
+ALTER TABLE discount_product_category
+    ALTER COLUMN product_category TYPE VARCHAR(255);
+
+UPDATE discount_product_category
+SET product_category = 'CULTURE_AND_ENTERTAINMENT'
+WHERE product_category IN ('ENTERTAINMENT', 'FOOD_DRINK', 'HOTELS', 'SHOPPING');
+
+UPDATE discount_product_category
+SET product_category = 'HOME'
+WHERE product_category = 'SERVICES';
+
+DROP TYPE IF EXISTS product_category_enum CASCADE;
```

Comment:
not sure this can me performed

Reply:
Tests are applying it correctly, what's your thought?

Reply:
maybe I' wrong but when trying to apply this in the past on existing data I've encountered several problems :)